geodiff/src/drivers/sqlitedriver.cpp

```diff
@@ -152,10 +163,10 @@ void SqliteDriver::create( const DriverParametersMap &conn, bool overwrite )
   }
 
   mDb = std::make_shared<Sqlite3Db>();
-  mDb->create( base );
+  mDb->create( context()->logger(), base );
 
   // register geopackage related functions in the newly created sqlite database
-  if ( !register_gpkg_extensions( mDb ) )
+  if ( !register_gpkg_extensions( context(), mDb ) )
```

Comment:
exception can be throws in the register_gpkg_extensions, so you do not need to pass context to the function and also catch false here?